stubs/common/Facades.stub

```diff
@@ -5,6 +5,7 @@ namespace Illuminate\Support\Facades;
 abstract class Facade {}
 
 /**
+ * @mixin \Illuminate\Redis\Connections\Connection
```

Comment:
Can we use `\Illuminate\Redis\RedisManager` instead?

Reply:
It's already there on the line 10.